src/Admin/AbstractAdmin.php

```diff
@@ -657,7 +690,13 @@ public function update($object)
     {
         $this->preUpdate($object);
         foreach ($this->extensions as $extension) {
+            // NEXT_MAJOR: check if extension implements PreUpdateInterface before calling the method
             $extension->preUpdate($this, $object);
```

Comment:
This looks wrong… shouldn't you check whether it's an old extension or it's an instance of `PreUpdateInterface`? What if it is new and doesn't implement `PreUpdateInterface`?

Reply:
you are right, will fix this.